config/application.yml

```diff
@@ -0,0 +1,17 @@
+defaults: &defaults
+  jwt:
+    key: <%= Rails.root.join('config', 'jwt', 'dev.pem') %>
```

Comment:
Okay, but - this being a private key - we do not want to commit `prod.pem` to Github. Should I modify `config/application.yml` to seek this file in `config/jwt` regardless?